transcom/mymove
# INT B-21023 fix typo on dependents included question

## [Agility ticket] [B-21023](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1006537&RoomContext=TeamRoom%3A1007470&concept=TeamRoom)

## Summary
This fixes a typo on the "Tell us about the orders" page. The text is change from "Are dependents included the orders?" to "Are the dependents included **in** the orders?" 

Additionally a test was added to verify the text on the screen.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the MilMove application.
2. Login as a Services Counselor.
3. Click on the Customer Search tab.
4. Search for a Customer Name.
5. In the results table, click on the Create Move button.
6. Scroll to the bottom of the Confirm Customer Info page and click the Save button.
7. On the Tell us about the orders page confirm the depends questions text is "Are dependents included in the orders?"

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots![image](https://github.com/user-attachments/assets/c85bc0fe-b536-4dac-aa63-daed7713c67b)
